event_webhook_types.go

```diff
@@ -38,6 +38,17 @@ const (
 	StateIDLocked StateID = 4
 )
 
+// CommentEventAction identifies if a comment has been newly created or updated.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#comment-events
+type CommentEventAction string
+
+const (
+	CommentEventActionCreate CommentEventAction = "create"
+	CommentEventActionUpdate CommentEventAction = "update"
+)
```

Comment:
Looks great, thanks! Can you move this snippet to the `types.go` file and insert it in its alphabetical order in the file?

Reply:
done 👍